test/extended/cmd.sh

```diff
@@ -143,3 +144,15 @@ os::cmd::expect_success "oc process -f test/extended/fixtures/image-pull-secrets
 os::cmd::try_until_text "oc get pods/my-dc-1-hook-pre -o jsonpath='{.status.containerStatuses[0].imageID}'" "docker"
 os::cmd::expect_success "oc delete all --all"
 os::cmd::expect_success "docker rmi -f ${docker_registry}/image-ns/busybox:latest"
+
+# check to make sure that service serving cert signing works correctly
```

Comment:
> test/extended/cmd is not in conformance and therefore not tested on a normal PR job

Wow, so we really do have bash scripts written in Go to call go code. It's like the worst of all possible worlds.

@smarterclayton can I get a reprieve until we inject the CA so I can use podB to validate podA.  Otherwise I'm hunting around trying to find bits of master config inside the test to find the CA certs in question and building an http handler by hand, all of which is a simple one-liner in bash.